01-atoms/text/Heading.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withModifiers from '../../_utils/withModifiers';
+
+const Heading = ({
+  base_class = 'heading',
```

Comment:
I don't think this is official, but I typically see props in camelCase. Doesn't matter to me as long as we are fairly consistent. Yep, I'm being nitpicky because everything else looks great. And yep, I didn't mention this in the last PR for things like button_modifiers.

Reply:
This is good. I'll go back and update everything to use camelcase

Reply:
Done